build.gradle

```diff
@@ -1,3 +1,3 @@
 allprojects {
-    version = "0.2.8"
+    version = "0.2.9"
```

Comment:
you should not up the version in this PR. Thats a separate concern that we can do afterwards :x: